clang/include/clang/Sema/Sema.h

```diff
@@ -6891,7 +6891,7 @@ class Sema final : public SemaBase {
       const TemplateArgumentListInfo *TemplateArgs);
 
   ExprResult ActOnMemberAccessExpr(Scope *S, Expr *Base, SourceLocation OpLoc,
-                                   tok::TokenKind OpKind, CXXScopeSpec &SS,
+                                   bool IsArrow, CXXScopeSpec &SS,
```

Comment:
You could land this refactoring straight away on an NFC commit, to clean up for the review a bit.